### PR TITLE
Update repo check

### PR DESCRIPTION
**Pull request by hoethan231**

### CODE REVIEW

File: sce.sh [hunk 1]
```diff
@@ -48,20 +48,36 @@ GITHUB_BASE_HTTP_URL="https://github.com/SCE-Development/"
...
+    ["d"]="SCE-discord-bot"
+)
+
+declare -A VALID_COMMANDS=(
+    ["link"]="help message"
+    ["clone"]="help message"
+    ["run"]="help message"
+    ["setup"]="help message"
+    ["completion"]="help message"
+    ["create"]="help message"
```
Comment (evanugarte) [inline sce.sh:79]: does this work
```suggestion
    ["link"]=1
    ["clone"]=1
    ["run"]=1
    ["setup"]=1
    ["completion"]=1
    ["create"]=1
```

File: sce.sh [hunk 2]
```diff
@@ -225,12 +187,12 @@ then
     then
         print_missing_config $REPO_LOCATION $missingPaths
     fi
-    if [ $start_only_mongodb_container -eq 0 ]
+    if [ $start_only_mongodb_container == 0 ]
```
Comment (evanugarte) [inline sce.sh:190]: lets make it so 0 means false, and 1 means true

if `start_only_mongodb_container` is 0, dont do it, if it is 1, start mongo only

File: sce.sh [hunk 3]
```diff
@@ -225,12 +187,12 @@ then
...
     then
         docker-compose -f docker-compose.dev.yml up mongodb -d
         exit 0
     fi
-    if [ $name == $SCE_DISCORD_BOT_REPO_NAME ]
+    if [ $name == "SCE-discord-bot" ]
```
Comment (evanugarte) [inline sce.sh:195]: undo?

Comment (hoethan231) [reply sce.sh:195]: I renamed all the repo variables into strings so this should be fine.

File: sce.sh (hunk 2 above)
Comment (hoethan231) [reply sce.sh:190]: I wanted to be consistent with whoever wrote it before and they used 0 to indicate a successful exit (in this case, yes for starting container). Do you still want me to update it?

**Review by evanugarte — APPROVED** [review]

File: sce.sh (hunk 2 above)
Comment (evanugarte) [reply sce.sh:190]: exit codes are like "did something bad happen?" 0 means no, any other code means yes, the code value being an indicator of what went wrong

in both cases, 0 means false, pls update when you can

Comment (hoethan231) [reply sce.sh]: done

**Pull request merged**